README.md

```diff
@@ -1,101 +1,12 @@
-
-## Testing
-Basic test boilerplate is included with [Jest](https://facebook.github.io/jest/) and
-[Enzyme](http://airbnb.io/enzyme/). Jest is a test runner, an assertion library and a snapshot tester, whereas Enzyme
-allows DOM testing. See the examples included in `__test__` to get an idea.
-
-### Continuous integration
-If you want CI and nice passing/failing badges, enable the repository in [Travis CI](https://travis-ci.org/). Add the
-file `.travis.yml` with the following:
-```
-language: node_js
-node_js:
-  - "8"
-```
-
-Now, with each push, Travis CI will run your tests and generate a report. You can display a test status badge going to
-Travis CI, clicking on the badge and pasting the Markdown embed snippet on your `README.md`.
-
-Enabling code coverage is very similar. You need to enable your repository in [Coveralls](https://coveralls.io/) and
-install the `coveralls` package:
-```
-npm install --save-dev coveralls
-```
-
-Add the `posttest` script to `package.json`:
-```
-"posttest": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
-```
-
-Now, every time that Jest is run, it will generate coverage information and send it to Coveralls for a coverage report.
-If you go to Coveralls, you can also get a snippet to embed the coverage report shield on your readme file.
+The left panel box introduces the experiments clustered by species, which uses [atlas-homepage-card](https://github.com/ebi-gene-expression-group/atlas-homepage-cards) package.
 
+The right panel box introduces featured experiments and latest experiments, which use different styled card components.
 
-## What’s included?
-- [React 16 and PropTypes](https://facebook.github.io/react/)
-- [URI.js](https://medialize.github.io/URI.js/) for URL manipulation
-- [Babel](https://babeljs.io/) with presets `env`, `react` and the [object spread operator plugin](https://babeljs.io/docs/plugins/transform-object-rest-spread/) (see `.babelrc`)
-- [Webpack 4 with Webpack-Command and Webpack-Serve](https://webpack.js.org/)
-- [Jest](https://facebook.github.io/jest/) and [Enzyme](http://airbnb.io/enzyme/) for testing
-
-## Polyfills
-No polyfills are included by default, but you might want one or both of these two:
-- [Fetch polyfill](https://github.com/github/fetch)
-- [Babel polyfill](https://babeljs.io/docs/usage/polyfill/)
-
-### NPM
-```
-npm install --save-dev whatwg-fetch babel-polyfill
-```
-
-Tweak your `webpack.config.js` to include them in your entry points:
-```
-entry: {
-  myComponent: [`babel-polyfill`, `whatwg-fetch`, `./html/render.js`]
-  ...
-}
-```
+The current version applies two new endpoints to fetch the latest experiments and a static landing page for HCA, which is implemented in a new [atlas](https://github.com/ebi-gene-expression-group/atlas/tree/feature/161508497-add-endpoint-for-latestExperiments-and-static-landingPage) branch
 
 ## Run it on your browser
 Use Webpack-Serve:
```

Comment:
Remember we switched to WDS now.

Reply:
What WDS means here? Actually I just deleted this line.